src/components/core/geoChart/GeoChartRenderer.tsx

```diff
@@ -269,7 +276,15 @@ export default class GeoChartRenderer extends React.Component<IGeoChartRendererP
         if (!this.chart) {
             return;
         }
-        this.chart.remove();
+        // try catch to hide the mapbox's error message
+        // TypeError: Cannot read property 'off' of undefined
+        // mapbox is trying to call its function after deleted
+        // https://github.com/mapbox/mapbox-gl-js/blob/master/src/ui/control/navigation_control.js#L118
+        try {
+            this.chart.remove();
+        } catch (_e) {
```

Comment:
should remove `(_e)`

Reply:
we can't.
When I remove that parameter, it said
`8 - error TS1003: Identifier expected.
285         } catch () {`

Reply:
should be:
```
try {
   this.chart.remove();
} catch { return; }
```